libmscore/chord.cpp

```diff
@@ -2897,7 +2897,7 @@ QPointF Chord::layoutArticulation(Articulation* a)
 
 void Chord::reset()
       {
-      score()->undoChangeProperty(this, P_ID::STEM_DIRECTION, Direction(Direction::AUTO));
+      score()->undoChangeProperty(this, P_ID::STEM_DIRECTION, Direction::AUTO);
```

Comment:
We need a QVariant of type Direction and not int. Direction is automatically converted to QVariant so QVariant a = Direction(Direction::AUTO) is ok, QVariant a = Direction::AUTO; not. QVariant a = int(Direction...) gives the wrong (int) QVariant.
Somehow i could not manage to get Direction::AUTO converted automatically to QVariant of type Direction. Its ambiguous somehow.